src/direct_commands/claim.ts

```diff
@@ -0,0 +1,34 @@
+import { ChannelType, Client, Message } from "discord.js"
+import config from "../lib/config"
+import Log from "../utils/logger"
+
+const claim = async (msg: Message, args: string[], bot: Client): Promise<Message<boolean>> => {
+  if (msg.channel.type !== ChannelType.DM) return msg.reply({ content: 'This command can only be used in DMs!' })
+  if (args.length < 2) return msg.reply({ content: 'Invalid format, please use !claim <ign> <code>' })
+  const untrustedGuild = await bot.guilds.fetch(config.untrusted_guild_id);
+  const member = await untrustedGuild.members.fetch(msg.author.id)
+  if (!member) return msg.reply({ content: 'You are not in the untrusted guild!' })
+  if (member.roles.cache.find(r => r.id === config.image_role_url()))
```

Comment:
This should be changed to r.id === config.image_role_id

Reply:
Fixed in recent commits -- needs re-review.